packages/terra-table/src/terra-dev-site/doc/table/guides.5/SingleSelectTable.1.doc.mdx

```diff
@@ -93,10 +93,10 @@ Finally we can implement a method to loop through our data and create the table
 +         checkLabel: 'Single Selection',
 +       },
 +       cells: [
```

Comment:
Should the summary have been updated?

Reply:
Currently summary gives interaction details about table so left it as it is.